tools/tkn-results/docs/tkn-results_logs_get.md

```diff
@@ -0,0 +1,29 @@
+## tkn-results logs get
+
+Get Record
+
+```
+tkn-results logs get [flags] <record>
+
+  <record parent>: Record parent name to query. This is typically "<namespace>/results/<result name>", but may vary depending on the API Server. "-" may be used as <result name> to query all Results for a given parent.
+```
```

Comment:
@khrm it looks like there is an auto-gen mismatch. The docs here don't match the code.

Reply:
Strange. I ran the command again.